lib/darwin_core/occurrence.rb

```diff
@@ -38,7 +38,7 @@ class Occurrence
       %w(recordedByID http://rs.tdwg.org/dwc/terms/recordedByID),
       %w(identifiedBy http://rs.tdwg.org/dwc/terms/identifiedBy),
       %w(identifiedByID http://rs.tdwg.org/dwc/terms/identifiedByID),
-      %w(captive https://www.inaturalist.org/schema/terms/captive),
+      %w(captive https://www.inaturalist.org/schema/terms/captive_cultivated),
```

Comment:
pretty minor - if we're already changing the URI, would it make sense to put it at `https://www.inaturalist.org/terms/captive_cultivated` (remove schema). I think this is the only place we reference `schema/terms` when using our own custom URIs. Both would resolve, but removing schema seems like it would be most consistent